Sources/HttpServer/Server/Server.swift

```diff
@@ -69,7 +70,7 @@ public class Server {
             return parsedRequest
         } catch let error {
             print (error.localizedDescription)
-            return HttpRequest(method: nil, url: "", version: "", headers: ["" : ""], body: ["": ""])
+            return HttpRequest(method: nil, url: "", params: [], version: "", headers: ["" : ""], body: ["": ""])
```

Comment:
Would it be possible to have a static method on `HttpReqest` that returns a totally empty request? Something like `HttpRequest.emptyRequest`?